bin/node/runtime/Cargo.toml

```diff
@@ -81,7 +81,7 @@ pallet-utility = { version = "2.0.0", default-features = false, path = "../../..
 pallet-transaction-payment = { version = "2.0.0", default-features = false, path = "../../../frame/transaction-payment" }
 pallet-transaction-payment-rpc-runtime-api = { version = "2.0.0", default-features = false, path = "../../../frame/transaction-payment/rpc/runtime-api/" }
 pallet-vesting = { version = "2.0.0", default-features = false, path = "../../../frame/vesting" }
-pallet-cere-ddc = { version = "3.0.0", default-features = false, path = "../../../frame/ddc-pallet" }
+pallet-cere-ddc = { version = "4.0.0", default-features = false, path = "../../../frame/ddc-pallet" }
```

Comment:
What was changed there? Where is the RP?

Reply:
https://github.com/Cerebellum-Network/ddc-pallet/pull/5